packages/webapp/src/components/Task/MovementTask/index.tsx

```diff
@@ -37,7 +38,8 @@ const PureMovementTask = (props: PureMovementTaskProps) => {
   const PURPOSE = `movement_task.purpose`;
   const OTHER_PURPOSE_EXPLANATION = `movement_task.other_purpose_explanation`;
 
-  const selectedPurpose = watch(PURPOSE);
+  const selectedPurposes: AnimalMovementPurpose[] & { value: number; label: string } =
```

Comment:
I think the type should be `(AnimalMovementPurpose & { value: number; label: string })[]` 😄

Regarding translation, I’ve noticed that others are still adding `MISSING`, so I was wondering! I guess I’ll be the first one to stop adding them 🤭

Reply:
It will update Crowdin correctly with just the english change! I have been doing the MISSING just as a reminder for ourselves and QA in app. Something searchable to make sure there is an update. If we decide it is not valuable I can stop too!

Reply:
🤦‍♀️ re the type -- I'm so glad you're reviewing because my mind is all over the place 😂